### PR TITLE
Fix: No route when bind_interface

### PR DESCRIPTION
**Pull request by chise0713**

### CODE REVIEW

File: proxy/wireguard/tun_linux.go [hunk 1]
```diff
@@ -199,6 +199,7 @@ func createKernelTun(localAddresses []netip.Addr, mtu int, handler promiscuousMo
 
 		r := netlink.NewRule()
 		r.Table, r.Family, r.Src = ipv6TableIndex, unix.AF_INET6, addr.IPNet
+		r.Table, r.Family, r.OifName = ipv6TableIndex, unix.AF_INET6, n
```
Comment (mmmray) [inline proxy/wireguard/tun_linux.go:202]:
```suggestion
		r.OifName = n
```

those other things are already assigned

File: proxy/wireguard/tun_linux.go [hunk 2]
```diff
@@ -199,6 +199,8 @@ func createKernelTun(localAddresses []netip.Addr, mtu int, handler promiscuousMo
 
 		r := netlink.NewRule()
 		r.Table, r.Family, r.Src = ipv6TableIndex, unix.AF_INET6, addr.IPNet
```
Comment (mmmray) [inline proxy/wireguard/tun_linux.go:201]: what happens to this rule? I think it is simply overwritten

Comment (chise0713) [reply proxy/wireguard/tun_linux.go:201]: I forget to `append()`, my fault.

Comment (chise0713) [reply proxy/wireguard/tun_linux.go:201]: But my local repository didn't, so maybe yesterday I was too sleepy so I forgot to Ctrl + S 🥲

**Pull request merged**